config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -319,6 +319,67 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
+- name: ci-kubernetes-kubemark-gce-scale-scheduler
+  tags:
+  - "perfDashPrefix: kubemark-5000Nodes-scheduler"
+  - "perfDashJobType: performance"
+  interval: 48h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+    preset-e2e-kubemark-gce-scale: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-kubemark
+    testgrid-tab-name: kubemark-5000
```

Comment:
Maybe `kubemark-5000-scheduler`? This tab is already claimed by `ci-kubernetes-kubemark-gce-scale`

Reply:
Done.